domains/event-store/poc/EventStore/Serialization/Event.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using EnergyOriginDateTimeExtension;
+
+namespace EventStore.Serialization;
+
+public class Event {
```

Comment:
We should consider using records for Event: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/record

Reply:
Done